### ISSUE TITLE
Inception resenet v2 stem replicates the stem of v1 instead of v2 as described in the original paper

### ISSUE THREAD

**Issue by agsourav** [Cadene/pretrained-models.pytorch#197]
The stem implemented in inception resenet v2 has replicated the stem of inception resnet v1. The two are however different in the original paper [https://arxiv.org/pdf/1602.07261.pdf](url)

**Comment by xinhui123** [comment]
yes, inception resnet v2 have the same stem as inception v4 according to the original paper. The Inception-resnet-A/B/C are also repeated for different number of times compared to the original paper.
